releasenotes/notes/removes-dd-prioritiy-sampling-config-5d07f4aedbfa4dc4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    tracing: Removes the `DD_PRIORITY_SAMPLING` configuration option. This option was deprecated in v1.19.0.
```

Comment:
```suggestion
    tracing: Removes the ``DD_PRIORITY_SAMPLING`` configuration option. This option was deprecated in v1.19.0.
```

Reply:
Do we instead want to state this as a deprecation and then state it already does nothing?

The more important thing to document is that the `priority_sampling=` arg to `Tracer.configure()` is being removed.